src/celeste/mergeSortedArrays.js

```diff
@@ -0,0 +1,35 @@
+// Merge Sorted Arrays (medium)
+//
+// You are given two arrays of numbers that are already sorted in to
+// ascending order.
+//
+// Write a function called mergeSortedArrays() that takes two sorted arrays
+// and merges them in to one array that is also sorted in ascending order.
+//
+// -----------------------------------------------------------------------
+
+function mergeSortedArrays(a, b) {
+  // make new arrays that i can fuck w/
+  let arrA = Array.from(a)
+  let arrB = Array.from(b)
```

Comment:
is it important your original argument arrays not be altered? I think it's fine to alter them so I don't think you need to use the space to create new variables and copies of the original arrays.